### ISSUE TITLE
Failed to export C# project

### ISSUE THREAD

**Issue by L2750558108** [godotengine/godot#70883]
### Godot version

3.5.1mono

### System information

windows10

### Issue description

![image](https://user-images.githubusercontent.com/119792950/210405439-82432c5d-87c6-45ec-93c8-59d44e00831f.png)

### Steps to reproduce

Create a project
Create a scene
Create a C# script and mount it to any node of any scene
Create an IOS export setting
Export it
Get this
![image](https://user-images.githubusercontent.com/119792950/210405815-6026541d-9115-44a4-9e2e-7e2e4acc1b6a.png)

### Minimal reproduction project

[test.zip](https://github.com/godotengine/godot/files/10338441/test.zip)

**Comment by Chaosus** [comment]
Not quite sure, but I think it's a duplicate of https://github.com/godotengine/godot/issues/44577.

**Comment by bruvzg** [comment]
> windows10

You can export Mono project for iOS from macOS only. AOT compiler is macOS app. And exported project build require Xcode.

**Comment by akien-mga** [comment]
I don't know how it behaves when attempting to export to iOS for a non-Mono build from Windows/Linux, but there's no reason for Mono to behave differently here, cross platform export to iOS is impossible regardless of the Godot flavour.

**Comment by bruvzg** [comment]
Non-Mono project can be partially exported, it will export Xcode project but won't build in and create `.ipa`. Not sure how useful it is, since it's necessary to move in to mac to finish the build anyway.

**Comment by L2750558108** [comment]
Is it possible to export a full Xcode project when trying to export a C# project from Windows to iOS?

Even if it can't export, does it make the error message a little clearer? The error message now is "Cannot find mono runtime executeable xxx". A better way to prompt is "Transfer to macOS to export for iOS", or it can directly display a red text prompt "Only macOS platform can export for iOS" in the export window interface
